frontend/ui/data/credefaults.lua

```diff
@@ -0,0 +1,19 @@
+return {
+    prefix = 'copt',
+    options = {
+        screen_mode = "portrait",
+        view_mode = 0,
+        render_dpi = 96,
+        line_spacing = DCREREADER_CONFIG_LINE_SPACE_PERCENT_MEDIUM,
+        page_margins = DCREREADER_CONFIG_MARGIN_SIZES_MEDIUM,
+        font_size = DCREREADER_CONFIG_DEFAULT_FONT_SIZE,
+        font_weight = 0,
+        font_gamma = 15,
+        font_hinting = 2,
+        font_kerning = 1,
```

Comment:
Is that "fast"? I think that "good" (cached HB) might make more sense as a default since it's almost just as fast and much better looking, just without ligatures.

Reply:
It's still "fast", and changing it should probably not be part of this PR :)
cf #4483 _(Not yet set as a new default instead of Freetype, let's use that new one for a while to be sure it's really good and deserves to be the default.)_

Reply:
True, I suppose that'd make it harder to track. I haven't noticed anything wrong in my regular use of "good" over the past few weeks, except the obvious that I prefer "best". ;-)